CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ FEATURES:
 ENHANCEMENTS:
 
 BUG FIXES:
-* Cusotm actions fail on resources with a pipline ([#3646](https://github.com/microsoft/AzureTRE/issues/3646))
+* Custom actions fail on resources with a pipeline ([#3646](https://github.com/microsoft/AzureTRE/issues/3646))
```

Comment:
@marrobi : This is a typo fix for a historic change log entry. Will anything/anyone get upset by this change? Nothing that we know/care about looks for changes in the change log, right?

Reply:
right